lib/player.rb

```diff
@@ -0,0 +1,56 @@
+class Player
+    attr_reader   :name, :token, :cells_chosen
+  
+    def initialize(name,selected,board)
+      @board = board
+      @selected_token = selected
+      @name = welcome_player(name)
```

Comment:
you can simply have used get_name(name) here. As mentioned below, the `welcome_player` method might be unnecessary